### ISSUE TITLE
WebView2 Download Uwp Panel Not Opening Download Folder - Latest Pre-Release

### ISSUE THREAD

**Issue by jarno9981** [MicrosoftEdge/WebView2Feedback#3343]
**Description**
when i download something from the internet with a embedded webview2 in uwp the download in the background fires but the panel where the downloads live will  open but the downloads dont get any folder or icon

**Version**
SDK: 1.0.1724-prerelease / On Older Same Issues Apply
Runtime: Edge Stable - 111.0.1661.54 / 64bit
Framework: UWP
OS: Win11 - 22624.1470

**Repro Steps**
make a blank uwp project open webview2 set any url than go to random download page click the download button and downloads start but downloads panel will show but no image or any kind
Some casis random freeze of the panel

https://user-images.githubusercontent.com/53493418/228745660-95633acc-6b49-4851-94cf-56c0f6e6ed9e.mp4

**Additional context**
Included - Bugy Repo : https://www.dropbox.com/s/vauijw3kkmt2nl6/buggy.zip?dl=1

**Comment by jarno9981** [comment]
@champnic  new issues openend

**Comment by champnic** [comment]
@jarno9981 Can you clarify what you mean by "panel where the downloads live will open but the downloads dont get any folder or icon"? In the video I don't see the panel open at all, but I also see the mouse disappear so I think that might be an issue with whatever screen capturing software you're using. Can you include a screenshot?

**Comment by jarno9981** [comment]
> @jarno9981 Can you clarify what you mean by "panel where the downloads live will open but the downloads dont get any folder or icon"? In the video I don't see the panel open at all, but I also see the mouse disappear so I think that might be an issue with whatever screen capturing software you're using. Can you include a screenshot?

I will do that later after work i will unclude screenshot

**Comment by jarno9981** [comment]
![Schermopname (8)](https://user-images.githubusercontent.com/53493418/229304115-11864655-e3e2-45d7-bab7-0f87ae2471ad.png)

**Comment by champnic** [comment]
Ah - I believe we had to remove that as it was broken in UWP because the WebView2 didn't have permissions to open the folder location in explorer. @maurawinstanley can you confirm?

**Comment by jarno9981** [comment]
> Ah - I believe we had to remove that as it was broken in UWP because the WebView2 didn't have permissions to open the folder location in explorer. @maurawinstanley can you confirm?

Any work around for it

**Comment by maurawinstanley** [comment]
Yes, this is by design right now. Workaround wise, I do not think there is a viable-workaround given there is no capability to declare full access to the Downloads folder right now.

**Comment by 17Amir17** [comment]
Is this being worked on?

**Comment by champnic** [comment]
Not currently, no.

**Comment by jarno9981** [comment]
> Not currently, no.
@champnic 
when will it be work on please as quick as possible its a big function

and whats more strange downloads work fine in WinUi3

**Comment by 17Amir17** [comment]
> and whats more strange downloads work fine in WinUi3

Yup also on wpf

**Comment by jarno9981** [comment]
> > and whats more strange downloads work fine in WinUi3
> 
> Yup also on wpf

So uwp is the only one with download problems

**Comment by champnic** [comment]
UWP is the only framework that imposes the extra restrictions on apps to access files and folders like Downloads. WinUI3/WinAppSDK doesn't currently support UWP.
